### PR TITLE
Javascript javascript1 week3

### PR DESCRIPTION
**Pull request by saloumeh-67**
homework for Javascript 1 Week 3 .

### CODE REVIEW

File: javascript/javascript1/week3/WhenWillWeBeThere.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function calculation() {
+    const timePeriod = travelInformation.destinationDistance / travelInformation.speed;
+    const minute = Math.floor((timePeriod % 1) * 60);
```
Comment (zuhairtaha) [inline javascript/javascript1/week3/WhenWillWeBeThere.js:7]: Why `%1` ?

File: javascript/javascript1/week3/itemArrayRemoval.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+ names.splice(names.indexOf(nameToRemove),1);
```
Comment (zuhairtaha) [inline javascript/javascript1/week3/itemArrayRemoval.js:12]: Nice 👍

File: javascript/javascript1/week3/notes.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+{
+let result;
+    for (let i = 0; i < notes.length; i++) 
+    {
+        if (notes[i].id === id) 
+        {
+            result = notes[i];
+            break;
+        } 
+        else if ((id === " ") || (id.typeof !== "number"))
```
Comment (zuhairtaha) [inline javascript/javascript1/week3/notes.js:23]: `id.typeof` !!?? 
Do you mean `typeof id`

File: javascript/javascript1/week3/notes.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            break;
+        } 
+        else if ((id === " ") || (id.typeof !== "number")) 
+        {
+            result = "invalid id";
+        }
+    }
+    return result;
+}
+firstNote = getNote(1);
```
Comment (zuhairtaha) [inline javascript/javascript1/week3/notes.js:30]: Declare the variable before using it

File: javascript/javascript1/week3/series.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  ];
+  function countHoursOfwatching() {
+    let sumOfTheTotalTime;
+    for (let i = 0; i < seriesDurations.length; i++) {
+        let currentDuration = 0;
+        let averageLife = 0;
+        averageLife = convertYearsToMinutes(seriesDurations[i].years);
+        currentDuration += (convertHoursToMinutes(seriesDurations[i].hours));
+        currentDuration += (convertDaysToMinutes(seriesDurations[i].days));
+        timeWastedInPercentage = (currentDuration / averageLife) * 100;
```
Comment (zuhairtaha) [inline javascript/javascript1/week3/series.js:29]: Declare this variable first. Add `const` / `let`

**Review by zuhairtaha — CHANGES_REQUESTED** [review]
Good job 👍

File: javascript/javascript1/week3/WhenWillWeBeThere.js (hunk 1 above)
Comment (saloumeh-67) [reply javascript/javascript1/week3/WhenWillWeBeThere.js:7]: conversion of time scale to decimal

**Pull request merged**